docs/en/restful_api.md

```diff
@@ -147,3 +147,12 @@ python -m lmdeploy.serve.gradio.app restful_api_url server_ip --restful_api True
 2. When OOM appeared at the server side, please reduce the number of `instance_num` when lanching the service.
 
 3. When the request with the same `instance_id` to `generate` got a empty return value and a negative `tokens`, please consider setting `sequence_start=false` for the second question and the same for the afterwards.
+
+4. Requests was processed one by one instead of on concurrency. If you got this, please set:
```

Comment:
I think you want to present `If you find requests are processed sequentially, rather than concurrently,`